### PR TITLE
Update docs to improve contribution setup

### PR DESCRIPTION
**Pull request by dtomcej**

### CODE REVIEW

File: .github/CONTRIBUTING.md [hunk 1]
```diff
@@ -33,32 +26,51 @@ $ ls dist/
...
+```
+$ export GOPATH=~/go
+$ export PATH=$PATH:$GOPATH/bin
+```
+
+This can be verified via `$ go env`
+- You will want to add those 2 export lines to your `.bashrc` or `.bash_profile`
+- You need `go-bindata` to be able to use `go generate` command (needed to build) : `$ go get github.com/jteeuwen/go-bindata/...` (Please note, the ellipses are required)
+
+###### Settig up your `glide` environment
```
Comment (emilevauge) [inline .github/CONTRIBUTING.md]: +n

File: .github/CONTRIBUTING.md [hunk 2]
```diff
@@ -33,32 +26,51 @@ $ ls dist/
...
+  (`go get …`) the dependencies in your `GOPATH`.
 - if you need another dependency, import and use it in
-  the source, and **run `glide get github.com/Masterminds/cookoo`** to save it in
+  the source, and run `$ glide get github.com/Masterminds/cookoo` to save it in
   `vendor` and add it to your `glide.yaml`.
 
 ```bash
 $ glide install
-# generate
+# generate (Only required to integrate other components such as web dashboard)
```
Comment (vdemeester) [inline .github/CONTRIBUTING.md]: Note that the script used inside the docker container still work here, `./script/make.sh binary` does both `generate` and `build`. I don't think we should mention it but :stuck_out_tongue_closed_eyes:

File: .github/CONTRIBUTING.md (hunk 1 above)
Comment (dtomcej) [inline .github/CONTRIBUTING.md]: Good catch!

**Comment by SantoDE** [comment]
Maybe just do a quick rebase to squash the second commit?

**Comment by dtomcej** [comment]
Ya, Sounds good. Will update the branch too.

**Comment by SantoDE** [comment]
LGTM now :) 👍

**Pull request merged**